node/src/main/scala/com/wavesplatform/block/Block.scala

```diff
@@ -38,7 +38,10 @@ case class Block(
 ) extends Signed {
   import Block._
 
-  val uniqueId: ByteStr = signature
+  lazy val uniqueId: ByteStr =
```

Comment:
How about
```suggestion
  val id: Coeval[ByteStr] =
```
for consistency with transactions and JSON representation?